tests/test_automatic_payout_transactions.py

```diff
@@ -8,8 +10,8 @@ def get_payouts():
     """
     # list of all data to return
     data = []
-    # api call of 1st page
-    stripe_obj = client["payouts"].list(limit=100, created={"gte": midnight})
+    # api call of 1st page starting from 4 days ago
```

Comment:
Please state the reason of it in comment

Reply:
Added the comment